daemon/graphdriver/devmapper/mount.go

```diff
@@ -59,6 +59,7 @@ func ProbeFsType(device string) (string, error) {
 	buffer := make([]byte, maxLen)
 	l, err := file.Read(buffer)
 	if err != nil {
+		file.Close()
```

Comment:
nope you just make changes on your local branch, `git add` those, then `git commit --amend --no-edit` and `git push -f your-remote checkfileoper1` and this PR will be updated automatically.